spec.bs

```diff
@@ -716,7 +716,54 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
       1. [=list/For each=] |ig| of |groups|:
         1. TODO: If rerunning `generateBid()` with only k-anonymous ads, remember to swap out |ig|'s ads
           to just k-anonymous ones so that [=validating an ad url=] excludes the non-k-anonymous ones.
-        1. TODO: Let |interestGroup| be ... from |ig| ... minus priority and prioritySignalsOverrides and any browser-defined pieces
+        1. Let |igIDL| be a new {{AuctionAdInterestGroup}}.
+        1. [=map/Set=] |igIDL|["{{AuctionAdInterestGroup/owner}}"] to |ig|'s [=interest group/owner=].
+        1. [=map/Set=] |igIDL|["{{AuctionAdInterestGroup/name}}"] to |ig|'s [=interest group/name=].
+        1. For each |interestGroupField| and |groupMember| in the following table <table class="data">
+            <thead><tr><th>Interest group field</th><th>Group member</th></tr></thead>
+            <tr>
+              <td>[=interest group/enable bidding signals prioritization=]</td>
+              <td>"{{AuctionAdInterestGroup/enableBiddingSignalsPrioritization}}"</td>
+            </tr>
+            <tr>
+              <td>[=interest group/execution mode=]</td>
+              <td>"{{AuctionAdInterestGroup/executionMode}}"</td>
+            </tr>
+            <tr>
+              <td>[=interest group/bidding url=]</td>
+              <td>"{{AuctionAdInterestGroup/biddingLogicURL}}"</td>

```

Comment:
"bidding url" is a [URL](https://url.spec.whatwg.org/#concept-url), but `biddingLogicURL` is a USVString. You can either explicitly serialize to a string, or you can create a [URL instance](https://url.spec.whatwg.org/#url-class) in the script's environment.